tests/exemple_rop.py

```diff
@@ -2,15 +2,15 @@
 
```

Comment:
No need to update this file, we can mark this example as obsolete.

Reply:
I find it sad to abandon this test. Does it work on 32 bits and python2?